libs/utils/utils.js

```diff
@@ -1295,3 +1297,32 @@ export function loadLana(options = {}) {
 }
 
 export const reloadPage = () => window.location.reload();
+
+export const userCanPublishPage = async (detail, isBulk = true) => {
```

Comment:
I don't think we should have code concerning authoring tools within the milo core utils.
The utils should be reserved for code that is meant for end users, it will end up on adobe.com and our end users are not involved in the publishing process, but currently would still load this code.

Reply:
this might be a good candidate for https://github.com/adobecom/milo/tree/stage/libs/tools/sharepoint or similar

Reply:
Okay sounds good, I will move it there